ecosystem/typescript/sdk_v2/src/client/core.ts

```diff
@@ -33,7 +33,7 @@ async function request<Req, Res>(
   const headers: Record<string, string | string[] | undefined> = {
     ...overrides?.HEADERS,
     "x-aptos-client": `aptos-ts-sdk/${VERSION}`,
-    "content-type": contentType ?? "application/json",
+    "content-type": contentType ?? MimeType.JSON.valueOf(),
```

Comment:
why we need the `valueOf`?

Reply:
Does it automatically convert to string?  I don't know how enums work in typescript necessarily

Reply:
I think having `MimeTime.JSON` is enough to output the string...?